package.json

```diff
@@ -103,7 +103,7 @@
     "release": "standard-version",
     "lint": "eslint .",
     "post-commit": "git status",
-    "type-check": "tsc --noEmit",
+    "type-check": "tsc --noEmit --project tsconfig.test.json",
```

Comment:
Is this intended?

Reply:
Yes, I want to check all files for TS errors, test files as well. Otherwise type issues only arrive when actually **running** the tests which is a bummer.

Reply:
The issue is that I want to be able to run my `yarn check` or `yarn type-check` command locally and be sure that all my code is correct. I don't want to have my tests catch a type error I could have fixed before sending to the CI or running the full checks on git hooks (which will fail my commit which i find very annoying ...)

Reply:
I see what you mean. I often consider test to be its own environment anyway (since it will be compiled and ran through e.g. `ts-node`). But either is fine by me 👍